Scribe/Components/InfoChildTableViewCell/InfoChildTableViewCell.swift

```diff
@@ -29,16 +34,31 @@ class InfoChildTableViewCell: UITableViewCell {
 
     return action
   }
+  
+  // MARK: - Functions
 
   func configureCell(for section: Section) {
     self.section = section
+
+    selectionStyle = .none
+
     titleLabel.text = section.sectionTitle
-    iconImageView.image = UIImage.availableIconImage(with: section.imageString)
+
+    if let shortDescription = section.shortDescription {
+      descriptionLabel.text = shortDescription
+
+      contentView.addSubview(descriptionLabel)
+    } else {
+      descriptionLabel.text = nil
+      descriptionLabel.removeFromSuperview()
+    }
 
     if !section.hasToggle {
+      accessoryType = .disclosureIndicator
       toggleSwitch.isHidden = true
     } else {
-      chevronImgView.isHidden = true
```

Comment:
Note to myself, remove the unused chevron image as we are now using the native one.